pkg/k8shandler/kibana/reconciler.go

```diff
@@ -28,6 +28,10 @@ import (
 const (
 	kibanaServiceAccountName     = "kibana"
 	kibanaOAuthRedirectReference = "{\"kind\":\"OAuthRedirectReference\",\"apiVersion\":\"v1\",\"reference\":{\"kind\":\"Route\",\"name\":\"kibana\"}}"
+	expectedCLOKind              = "ClusterLogging"
+	expectedCLOName              = "instance"
+	expectedCLOKibana            = "kibana"
+	expectedCLONamespace         = "cluster-logging"
```

Comment:
Using `cluster-logging` as expected namespace, typo?

Reply:
:facepalm: i do that all the time.. thank you